.github/workflows/withdraw-packages.yaml

```diff
@@ -0,0 +1,50 @@
+name: Withdraw packages
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Withdraw packages
+    runs-on: ubuntu-latest
+
+    permissions:
+      packages: write
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: chainguard-dev/actions/setup-melange@main
+
+      - uses: google-github-actions/setup-gcloud@v0
+
+      - run: echo "${{ secrets.MELANGE_RSA }}" > ./wolfi-signing.rsa
+      - run: |
+          sudo mkdir -p /etc/apk/keys
+          sudo cp ./wolfi-signing.rsa.pub /etc/apk/keys/wolfi-signing.rsa.pub
+
+      - name: 'Delete withdrawn packages'
+        run: |
+          for arch in x86_64 aarch64; do
+            for pkg in $(cat withdrawn-packages.txt); do
+              echo "=> $pkg"
+              gsutil -m rm gs://wolfi-production-registry-destination/os/$arch/$pkg
+            done
+          done
+
+      - name: 'Sync public package repository'
+        run: |
+          mkdir "${{ github.workspace }}/packages"
+          gsutil -m rsync -r gs://wolfi-production-registry-destination/os/ "${{ github.workspace }}/packages/"
+          find "${{ github.workspace }}/packages" -print -exec touch \{} \;
+
+      - name: 'Reconcile Wolfi index'
+        run: |
+          for arch in x86_64 aarch64; do
+            pushd "${{ github.workspace }}/packages/"$arch
+              melange index -o APKINDEX.tar.gz *.apk
+              melange sign-index --signing-key="${{ github.workspace }}/wolfi-signing.rsa" APKINDEX.tar.gz
+              gsutil -m rsync -r "${{ github.workspace }}/packages/${arch}/APKINDEX.tar.gz" gs://wolfi-production-registry-destination/os/${arch}/APKINDEX.tar.gz
```

Comment:
I think we can just `gsutil cp APKINDEX.tar.gz` instead of rsync, which feels easier to footgun into writing other stuff we don't want.